source/docs/articles/drupal.md

```diff
@@ -25,14 +29,14 @@ See [Updating Modules Through Drupal](/docs/articles/drupal/updating-modules-thr
 ## Apache Solr
 [Apache Solr](/docs/articles/sites/apache-solr/) is an open source system for indexing and searching website content. We manage a cluster of fast Solr index servers that communicate with your site via these popular Drupal modules:
 
-​[Apache Solr](https://drupal.org/project/apachesolr) - 7.x-1.4 and 6.x-1.8  
-[Search API Solr](https://drupal.org/project/search_api_solr) - 7.x-1.2
+​[Apache Solr](https://drupal.org/project/apachesolr) - 7.x-1.7 and 6.x-3.0  
```

Comment:
Is there a reason we're listing specific releases instead of minor versions? consider changing to `7.x-1.x` etc.